### PR TITLE
Renamed to swaggerize-routes, extensions, security.

### PR DESCRIPTION
**Pull request by tlivings**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+### 1.0.0 (renamed to swaggerize-routes)
```
Comment (aredridel) [inline CHANGELOG.md]: Gonna `npm deprecate` the old one?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+- Alternately specify `x-handler` properties to reference handlers at either the operation or path level.
```
Comment (aredridel) [inline CHANGELOG.md]: This isn't clear from this, though the below explanation works.

File: README.md [hunk 3]
```diff
@@ -78,6 +78,38 @@ handlers
...
+
+An alternative to automatically determining handlers based on a directory structure, handlers can be specified for both paths and/or operations.
+
+Example:
+
+```json
+{
+
+    "/pets": {
+        "x-handler": "handlers/pets.js"
```
Comment (aredridel) [inline README.md]: What is this path relative to? File or some project root or ?

File: README.md [hunk 4]
```diff
@@ -78,6 +78,38 @@ handlers
...
+
+    "/pets": {
+        "GET": {
+            "x-handler": "handlers/pets.js"
+        }
+    }
+}
+```
+
+These paths are relative to the `options.basedir` and are used as fallbacks for missing handlers from directory scan.
```
Comment (aredridel) [inline README.md]: Ah, cool. Nodey might specify them as './` but makes sense.

File: CHANGELOG.md (hunk 1 above)
Comment (tlivings) [inline CHANGELOG.md]: yes good call!

**Comment by aredridel** [comment]
All in all, :+1:

**Comment by pvenkatakrishnan** [comment]
not familiar with swagger spec as much, the code changes look good to me overall :+1:

**Pull request merged**